src/Compilers/CSharp/Portable/Binder/Binder_Crefs.cs

```diff
@@ -933,6 +933,12 @@ private ImmutableArray<ParameterSymbol> BindCrefParameters(BaseCrefParameterList
             foreach (CrefParameterSyntax parameter in parameterListSyntax.Parameters)
             {
                 RefKind refKind = parameter.RefKindKeyword.Kind().GetRefKind();
+                Debug.Assert(parameter.ReadOnlyKeyword.IsKind(SyntaxKind.None) ||
+                    (parameter.ReadOnlyKeyword.IsKind(SyntaxKind.ReadOnlyKeyword) && refKind == RefKind.Ref));
```

Comment:
We cannot assert that because the node can be created manually with invalid data. But we can still not check for those situations during binding. In other words, the `if` condition below is fine.